schema.sql

```diff
@@ -0,0 +1,12 @@
+CREATE DATABASE vet_clinic;
+\c vet_clinic;
+CREATE TABLE animals (
+  id INT PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(200) NOT NULL,
+  date_of_birth DATE NOT NULL, 
+  escape_attempts INT NOT NULL, 
+  neutered BIT NOT NULL,
```

Comment:
- [ ] It'll be good to use `BOOLEAN` as the data type here because it allows the entry of true/false values while you can store only 0/1 or NULL in a `BIT` data type.